geth/api/backend.go

```diff
@@ -193,11 +192,11 @@ func (b *StatusBackend) CallRPC(inputJSON string) string {
 }
 
 // SendTransaction creates a new transaction and waits until it's complete.
-func (b *StatusBackend) SendTransaction(ctx context.Context, args common.SendTxArgs) (hash gethcommon.Hash, err error) {
+func (b *StatusBackend) SendTransaction(ctx context.Context, args transactions.SendTxArgs) (hash gethcommon.Hash, err error) {
 	if ctx == nil {
 		ctx = context.Background()
 	}
-	tx := common.CreateTransaction(ctx, args)
+	tx := transactions.CreateTransaction(ctx, args)
```

Comment:
Let's double check exported methods and variables naming after moving from `common`.
For example, `transactions.CreateTransaction()` should look much clearer as `transactions.Create()`.

Reply:
@divan simplified exported methods I've moved from `common` package. 

There is also some margin for improvement on pre-existing transactions methods/vars naming, but imho, may be better to refactor them on a different iteration.

Reply:
Shouldn't it be `New` then?
`errors.New("blah")`, `transactions.New("blah")` looks more consistent with stdlib.

Reply:
Yep, but since we moved `transactions/queue` package inside `transactions` this package is managing different "objects":

`transactions.New` results on a new `TxQueue`
while:
`transactions.NewManager` on a new `Manager`

Reply:
hmmm... as far as i can see `transactions.New` is not being used outside of transactions package. I'll change that function name to something like `newQueue` and rename `transactions.NewManager` to `transactions.New` :)